src/main/java/seedu/anilist/logic/LogicManager.java

```diff
@@ -33,6 +36,7 @@
      * Constructs a {@code LogicManager} with the given {@code Model} and {@code Storage}.
      */
     public LogicManager(Model model, Storage storage) {
+        this.lastCmd = null;
```

Comment:
Purely design choice here, but would you consider using Optional<Command> instead?